pkg/controller/datavolume-controller.go

```diff
@@ -1397,6 +1397,10 @@ func (r *DatavolumeReconciler) reconcileProgressUpdate(datavolume *cdiv1.DataVol
 	}
 	pod, err := r.getPodFromPvc(podNamespace, pvcUID)
 	if err == nil {
+		if pod.Status.Phase != corev1.PodRunning {
+			// Avoid long timeouts and error traces from HTTP get when pod is already gone
+			return reconcile.Result{}, errors.New("Pod is not running, can't get metrics")
```

Comment:
I would just return reconcile.Result{}, nil instead of an error. We don't want to reconcile again because of this, so returning nil will not trigger an immediate reconcile.

Reply:
Done.